.github/workflows/chart-test.yaml

```diff
@@ -46,6 +49,9 @@ jobs:
         run: ct lint --target-branch main --all --chart-dirs charts --chart-dirs other-charts
         continue-on-error: true
 
+      - name: Remove ci symlink
+        run: rm charts/rstudio-workbench/ci
+
```

Comment:
I don't think this is needed. I believe that there are `post` steps that implicitly run that do this. Try without and check.